modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss

```diff
@@ -1,20 +1,16 @@
-.policy-detail-container {
-	.policy-detail-title::after {
-		background-color: var(--primary);
-		content: '';
-		display: block;
-		height: 1px;
-		margin-top: 5px;
-		position: relative;
-		width: 25px;
-	}
+.box {
+	max-height: 0;
+	opacity: 0;
+	transition: all 0.5s;
+}
```

Comment:
Is it from the .policy-detail-container?

Reply:
No, it's from the Panel Component. I will create a SCSS for the component.